src/screens/RequestsDetails/index.tsx

```diff
@@ -57,6 +57,38 @@ const RequestsDetails = ({ route: { params } }: RequestsDetailsProps): JSX.Eleme
     }, [id],
   );
 
+  const navigate = () => (type === 'technology'
+    ? navigation.navigate('Technology', {
+      data: {
+        id: technology.id,
+        title: technology.title,
+        slug: technology.slug,
+        image: technology.thumbnail?.url,
+        description: technology.description,
+        price: technology.costs.length ? technology.costs[0].price : 0,
+        createdAt: technology.created_at,
+        isSeller: !!(technology.costs.length && technology.costs[0].is_seller === 1),
+        type: 'technology',
+        terms: technology.terms,
+      },
+      type,
+    })
+    : navigation.navigate('Technology', {
```

Comment:
Later, we will have to change the name of this route to be more semantic like _"Solution"_ 🤓

Reply:
I agree 😅